.gitignore

```diff
@@ -46,3 +46,7 @@ nbproject
 # misc
 ############################
 dump.rdb
+
+\.vscode/

```

Comment:
@luislobo brought that into the templates in https://github.com/balderdashy/sails-generate/commit/f88aea6ceccfc92fb0c2fb8781046a88957e179c and https://github.com/balderdashy/sails-generate/commit/3a4de649a2266a130a4c57bb907dafe25d2cec21 and https://github.com/balderdashy/sails-generate/commit/9e34a044d9ae3c4e53f40a97d02a43feb30a6fbe and https://github.com/balderdashy/sails-generate/commit/811659bfc0187fe92360f98dc223ed570a67782d

So once that's published, gitignore files created with `sails new` or `sails generate etc` get these two things included by default

Reply:
Reverted back